packages/node_modules/@ciscospark/internal-plugin-lyra/test/integration/spec/space.js

```diff
@@ -18,6 +18,7 @@ describe('plugin-lyra', function () {
   describe('Space', () => {
     let participants;
     let lyraMachine;
```

Comment:
There are several other tests that still refer to `lyraMachine.space` ([example](https://github.com/webex/spark-js-sdk/blob/5212e5277847272f53f5265c03e1ec22d83b0ca6/packages/node_modules/%40ciscospark/internal-plugin-lyra/test/integration/spec/space.js#L90)). Should these be updated to the new `lyraSpace`?

Reply:
No those are not needed to be changed. lyraSpace just has the URIs, however the lyraMachine.space has the complete URLs including the lyra endpoint details. I have not changed the functions like putAudioState() etc which depends on complete lyra URLs and hence would need lyraMachine.space to be passed.